### PR TITLE
Make sure to copy use_tagging.

### LINKED ISSUE #30: Notice: Undefined index: use_tagging
I get the following notice thrown in

Cache\CacheBundle\Factory\DoctrineBridgeFactory

But I’m not sure where to put this in the config to fix it.

### PR DESCRIPTION
**Pull request by Nyholm**
This will fix https://github.com/php-cache/issues/issues/30

### CODE REVIEW

File: src/Command/CacheFlushCommand.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ protected function configure()
         $this->setDescription('Flushes the given cache');
         $this->addArgument('type', InputArgument::OPTIONAL, sprintf('Which type of cache do you want to clear? Valid types are: %s', implode(', ', self::VALID_TYPES)));
         $this->addArgument('service', InputArgument::OPTIONAL, 'If using type "provider" you must give a service id for the cache you want to clear.');
-        $this->setHelp(<<<EOD
+        $this->setHelp(<<<'EOD'
```
Comment (cryptiklemur) [inline src/Command/CacheFlushCommand.php:40]: I don't think this is correct?

Comment (Nyholm) [inline src/Command/CacheFlushCommand.php:40]: That is from styleCI. It surprised me too.

Comment (Nyholm) [inline src/Command/CacheFlushCommand.php:40]: https://styleci.io/analyses/qx0gry

Comment (cryptiklemur) [inline src/Command/CacheFlushCommand.php:40]: Weird...

**Pull request merged**